### DISCUSSION TITLE
No posts being returned on production, despite showing entries in the CMS UI.

### DISCUSSION THREAD

**Discussion by benslv** [Thinkmill/keystatic#1139] [Q&A] [answered]
Recently added Keystatic to my new [personal website](https://benslv.dev), written using Remix and hosted on Vercel.

I've hooked it up to GitHub rather than using the `local` storage option and am running into an issue when interacting with the site in production. No posts show up when I visit the [posts page](https://benslv.dev/posts), however I'm able to see all of the entries in the Keystatic UI completely fine?

They appear without issue when I run the same code locally, so I'm having trouble figuring out what I've done wrong. I'm using the same set of environment variables locally as I am in Vercel.

I've logged the Reader that is created in Remix's loader:
```
{
  collections: {
    posts: {
      read: [Function: read],
      readOrThrow: [AsyncFunction: readOrThrow],
      all: [AsyncFunction: all],
      list: [Function: list]
    }
  },
  singletons: {},
  repoPath: '/var/task',
  config: {
    storage: { kind: 'github', repo: [Object] },
    collections: { posts: [Object] }
  }
}
```

The `repoPath` looks a little weird to me but maybe that's correct?

**Comment by Surinder-thakur** [comment]
@benslv did you find your answer?

**Reply by benslv** [reply, marked as answer]
> @benslv did you find your answer?

I believe I fixed it with [this](https://bensilverman.co.uk/posts/how-to-make-vercel-play-nice-with-remix-and-keystatic/)!

If you console.log the directory you're trying to read, it seems to  make it work.
